src/layouts/Login/LoginPage.tsx

```diff
@@ -219,16 +230,39 @@ export const LoginPage = (): JSX.Element => (
     <RestrictedGovtMasthead />
     <Grid {...LOGIN_GRID_LAYOUT}>
       <GridItem area="image" bgColor="primary.500">
-        <Flex h="100%" w="100%" alignItems="end" justifyContent="center">
-          <LoginImage maxH="100%" />
+        <Flex
+          h="calc(100vh - 5rem - 2rem)"
+          w="100%"
+          alignItems="end"
+          justifyContent="start"
+          position="relative"
+        >
+          <Text
+            fontSize="2.25rem"
+            color="base.content.inverse"
+            textStyle="h2"
+            position="absolute"
+            maxW="14.25rem"
+            top="3rem"
+            right="5.5rem"
+          >
+            Rapidly build & launch informational sites
+          </Text>
+          <LoginImage maxH="100%" maxW="100%" />
         </Flex>
       </GridItem>
       <GridItem area="content" bgColor="white">
-        <Flex h="100%" maxW="54rem" alignItems="center" justifyContent="center">
+        <Flex
+          h="100%"
+          maxW="54rem"
+          alignItems="start"
+          justifyContent="center"
+          pt="3rem"
+        >
           <LoginContent />
         </Flex>
       </GridItem>
-      <GridItem area="credits" bgColor="primary.100">
+      <GridItem area="credits" bgColor="#E9F0FB" borderColor="#E9F0FB">
```

Comment:
can we have these as tokens?

Reply:
Hmm this is actually just a one off colour to match the stairs colour (the way this is being done is a little janky...), do we still want a token for it?

Reply:
ok then please ignore - just add comment bah

Reply:
2588e443